release-notes/aws/v9.3.2.md

```diff
@@ -0,0 +1,9 @@
+# :zap: Giant Swarm Release v9.3.2 for AWS :zap:
+
+This release fixes an issue where upgrading from an earlier platform release to platform release v9.3.0 or v9.3.1 would cause the `nginx-ingress-controller` service to have no endpoints.
```

Comment:
```suggestion
This release fixes an issue where upgrading from an earlier platform release to platform release v9.3.0 or v9.3.1 would cause the `nginx-ingress-controller` service to have no endpoints, causing the NGINX IC app not to work.
```

Reply:
Reviewed what we said [slack](https://gigantic.slack.com/archives/C02521EC0/p1589990471333600) to customers when we reported issue to make sure it connects